### DISCUSSION TITLE
nomad/1.0_admin/config_options

### DISCUSSION THREAD

**Discussion by giscus** [HCL-TECH-SOFTWARE/HCLSoftware-Documentation-Home#50] [Q&A]
# nomad/1.0_admin/config_options

Configure TLS using CertMgr on Domino Follow the instructions in the Domino documentation for setting up a manual or Let's Encrypt certificate for the Domino server. Ensure that the document for the ...

https://help.hcltechsw.com/nomad/1.0_admin/config_options.html

**Comment by kbmsg** [comment]
What is up with this deploy.nsf?
Which template does it rely on?
What do you mean we don't need it, and we should use safelinx setup?
I am trying to get NOMAD setup with SAML and trying to figure out this deploy.nsf thing.
